src/Components/Authentication/Authentication.js

```diff
@@ -5,14 +5,17 @@ import useUser from '../../hooks/useUser';
 import Loading from '../LoadingState/Loading';
 import Unavailable from '@redhat-cloud-services/frontend-components/Unavailable';
 import propTypes from 'prop-types';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+
 const Authentication = ({ children }) => {
   const queryClient = useQueryClient();
   const location = useLocation();
+  const chrome = useChrome();
 
   const { isLoading, isFetching, isSuccess, isError } = useUser();
 
   useEffect(() => {
-    isSuccess && window.insights?.chrome?.hideGlobalFilter();
+    isSuccess && chrome?.hideGlobalFilter();
```

Comment:
nit-pick: this should add more bit of safety by preventing calling `undefined` as a func if `hideGlobalFilter` is not defined
```suggestion
    isSuccess && chrome?.hideGlobalFilter?.();
```